Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -155,51 +156,121 @@ def add_user_to_departing_employee(self, username, departure_date=None, note=Non
             return None
         return user_id
 
-    def fetch_alerts(self, start_time, event_severity_filter):
+    def remove_user_from_departing_employee(self, username):
         try:
-            query = _create_alert_query(event_severity_filter, start_time)
-            res = self._sdk.alerts.search(query)
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.departing_employee.remove(user_id)
         except Exception:
             return None
-        return res["alerts"]
+        return user_id
 
-    def get_alert_details(self, alert_id):
+    def get_all_departing_employees(self):
         try:
-            res = self._sdk.alerts.get_details(alert_id)
+            res = []
+            pages = self._sdk.detectionlists.departing_employee.get_all()
+            for page in pages:
+                employees = page["items"]
+                res.extend(employees)
         except Exception:
             return None
-        return res["alerts"][0]
+        return res
 
-    def get_current_user(self):
+    def add_user_to_high_risk_employee(self, username, note=None):
         try:
-            res = self._sdk.users.get_current()
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.high_risk_employee.add(user_id)
+            if note:
+                self._sdk.detectionlists.update_user_notes(user_id, note)
         except Exception:
             return None
-        return res
+        return user_id
 
-    def remove_user_from_departing_employee(self, username):
+    def remove_user_from_high_risk_employee(self, username):
         try:
             user_id = self.get_user_id(username)
-            self._sdk.detectionlists.departing_employee.remove(user_id)
+            self._sdk.detectionlists.high_risk_employee.remove(user_id)
         except Exception:
             return None
         return user_id
 
+    def add_user_risk_tags(self, username, risk_tags):
+        try:
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.add_user_risk_tags(user_id, risk_tags)
+        except Exception:
+            return None
+        return user_id
+
+    def remove_user_risk_tags(self, username, risk_tags):
+        try:
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.remove_user_risk_tags(user_id, risk_tags)
+        except Exception:
```

Comment:
Is swallowing all exceptions an expected pattern? Do we have the capability to log info re: the exception somewhere instead of just silently failing?